### ISSUE TITLE
Lab 1 - Still unworkable

### ISSUE THREAD

**Issue by edbakermct** [MicrosoftLearning/AZ-104-MicrosoftAzureAdministrator#854]
Please close this - apparently it is godeploy using an old edition of the l;ab.

#The changelog states that all labs hjave been revised and tested BUT the tenant creation is still not possible due to MS / Shared subscription issues.

The lab is not usable at all.

There is a guided walkthrough for studwent sinstead.

I cannot stress how bad this looks - Lab 1 day sorry folks we cant do that due to Azure...

go watch this video and click some buttons.

Why doesnt the changelog state this is still the case? 

Even github does not mention this...

Cheers - diappointed AGAIN

Ed

**Comment by renaudwangler** [comment]
Hi, it's not that hard to bring a positive message to student about the  simulation and the fact that lab steps are here for those who'd like to experiment it in another environment of they own. I feel it's my job as a trainer to lead the group and deliver a positive message ;)
I had no bad feeling among students last week while teaching the course with the new labs (just avoid to "compare" new and old behaviour). Hope I helped ;)

**Comment by staleycyn** [comment]
What is the issue here? Is Go Deploy just showing the interactive lab simulation? Have you contacted their support desk and asked why they aren't doing the lab steps?

**Comment by edbakermct** [comment]
You cannot do the lab steps Microsoft have diabled the ability to create an entra id directory... so the lab is unusable on any platform as far sas i can tell. - i only have access to go deploy at present.

i even created a trial subscription and that is not allowed either - as i recal you had to have a PAID subscription to something on the account to be able to create a new tenant.

i cant remember as I havent taught this for a while but was hit by it in December.

My point was if the wissue isnt fixed why is the lab still here with no comment on github about not being able to use it.

cheers

ed

**Comment by staleycyn** [comment]
The labs were replaced 23 February. There is no tenant creation in the new lab. Is Go Deploy still using the old lab? The lab works fine for self-paced users and in another lab hoster that I have access to. This is the first complaint we have had about the lab not working.

**Comment by edbakermct** [comment]
Great thanks - will let go deploy know they are using the old lab.

they should be a great deal more switced on than that - 

that was the point of me raising it - i knew the changelog said they had all been re written.

**Comment by edbakermct** [comment]
hey Cindy i just got this response from godeploy - they are saying they are right - so what is actually going on then?

Lewis Hand commented:
Good afternoon,
Sorry to hear you are facing difficulties.
As per the launch information, this is a change that was made by Microsoft some time ago and is directly affecting the lab steps, with the new or older version of this lab.
There is unfortunately no work around for this at current and we are waiting on Microsoft and their content author to update the labs to resolve the problem.
I would advise skipping this lab and returning to it at a later date.
Kind regards,
Lewis
________________________________________

**Comment by staleycyn** [comment]
This has been escalated to the lab team that works with the Authorized Lab Hosters. There are no plans to update that lab, aside from minor fixes that are logged.

**Comment by kramit** [comment]
> i even created a trial subscription and that is not allowed either - as i recal you had to have a PAID subscription to something on the account to be able to create a new tenant.

Microsoft changed the way this worked a number of months ago now. You need P2 and they restricted the issuance of P2 and restricted Azure Passes for learning providers because of abuse of the system. So no, you wont be able to do it with a trial. Strange you have only managed to encounter it now. 

It is certainly not to do with out of date steps in a single lab provider, as it is an issue that effected all. 

At least there is an interactive guide to work around this. Which, is not the best solution at all, as Microsoft really should be on-top of this course more with testing before going live, as I can see from the fire-storm of errors here over the last few weeks. 

I just demo this with my own environments in the class to make sure it is covered in a better way for students than the pre-recorded click through.

**Comment by staleycyn** [comment]
Again, there are no lab instructions to create a tenant. Creating a tenant is not part of the AZ-104 certification. If there is something that needs to be fixed, please submit an Issue.
